### PR TITLE
Release/168.0.0

### PR DESCRIPTION
**Pull request by Prithpal-Sooriya**
## Explanation

This PR create a new release for

- `@metamask/profile-sync-controller` to `^0.1.1`
- `@metamask/notification-services-controller` to `^0.1.1`

## References

Fixes mobile integration issues.

###  `@metamask/profile-sync-controller`

- **CHANGED**: Tidied up and small refactors to support mobile integration ([#4441](https://github.com/MetaMask/core/pull/4441))

### `@metamask/notification-services-controller`

- **CHANGED**: Tidied up and small refactors to support mobile integration ([#4441](https://github.com/MetaMask/core/pull/4441))

## Checklist

- [x] I've updated the test suite for new or updated code as appropriate
- [x] I've updated documentation (JSDoc, Markdown, etc.) for new or updated code as appropriate
- [x] I've highlighted breaking changes using the "BREAKING" category above as appropriate

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@metamask/core-monorepo",
-  "version": "167.0.0",
+  "version": "168.0.0",
```
Comment (Prithpal-Sooriya) [inline package.json:3]: Whoops, I forgot to update this (was lost when I was reverting files).
Here was the original PR: https://github.com/MetaMask/core/pull/4469

**Review by Jonathansoufer — APPROVED** [review]
LGTM

**Comment by mcmire** [comment]
Hi @Prithpal-Sooriya. I'm afraid this won't work — you need to create a PR that reverts the original PR #4469, so that all of the packages you want to released have their previous version. Then you can bring back that PR and make sure to bump the root package version.

**Comment by Prithpal-Sooriya** [comment]
> Hi @Prithpal-Sooriya. I'm afraid that simply bumping the root package in this PR won't work — you need to create a PR that reverts the original PR #4469, so that all of the packages you want to released have their previous version. Then you can bring back that PR and make sure to bump the root package version. Otherwise the packages you want to release won't get released (because they're not being bumped in this PR, the release process will think that they've already been published).

Yep fair enough, I'll revert the release and re-open, apologies!

**Pull request closed without merging**